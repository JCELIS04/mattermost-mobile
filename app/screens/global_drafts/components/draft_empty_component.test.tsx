// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.

import React from 'react';

import {renderWithIntlAndTheme} from '@test/intl-test-helper';

import DraftEmptyComponent from './draft_empty_component';

describe('Draft Empty Component', () => {
    it('should match the snapshot', () => {
        const wrapper = renderWithIntlAndTheme(
            <DraftEmptyComponent/>,
        );
        expect(wrapper.toJSON()).toMatchSnapshot();
    });

    it('should render empty draft message', () => {
        const wrapper = renderWithIntlAndTheme(
            <DraftEmptyComponent/>,
        );
        expect(wrapper.getByText('No drafts at the moment')).toBeTruthy();
        expect(wrapper.getByText('Any message you have started will show here.')).toBeTruthy();
        expect(wrapper.toJSON()).toMatchSnapshot();
    });
});
