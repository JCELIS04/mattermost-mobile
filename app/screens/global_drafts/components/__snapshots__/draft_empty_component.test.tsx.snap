// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Draft Empty Component should match the snapshot 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
  testID="draft_empty_component"
>
  <Image
    source={1}
  />
  <Text
    style={
      {
        "color": "#3f4350",
        "fontFamily": "Metropolis-SemiBold",
        "fontSize": 20,
        "fontWeight": "600",
        "lineHeight": 28,
      }
    }
  >
    No drafts at the moment
  </Text>
  <Text
    style={
      {
        "color": "rgba(63,67,80,0.72)",
        "fontFamily": "OpenSans",
        "fontSize": 14,
        "fontWeight": "400",
        "lineHeight": 20,
        "marginTop": 8,
        "textAlign": "center",
      }
    }
  >
    Any message you have started will show here.
  </Text>
</View>
`;

exports[`Draft Empty Component should render empty draft message 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
  testID="draft_empty_component"
>
  <Image
    source={1}
  />
  <Text
    style={
      {
        "color": "#3f4350",
        "fontFamily": "Metropolis-SemiBold",
        "fontSize": 20,
        "fontWeight": "600",
        "lineHeight": 28,
      }
    }
  >
    No drafts at the moment
  </Text>
  <Text
    style={
      {
        "color": "rgba(63,67,80,0.72)",
        "fontFamily": "OpenSans",
        "fontSize": 14,
        "fontWeight": "400",
        "lineHeight": 20,
        "marginTop": 8,
        "textAlign": "center",
      }
    }
  >
    Any message you have started will show here.
  </Text>
</View>
`;
